docs/Language Feature Status.md

```diff
@@ -0,0 +1,31 @@
+# Language Feature Status
```

Comment:
There is no hint in this document that its intent is to describe the work of one particular team at one particular time. Moreover, it includes features that are currently *not* being worked on by the compiler team, such as `out var`, and it fails to include the non-compiler-team contributions that we have on our plate (#5050 const var, #8594 expression body accessors). You should clarify that, and note that the list will be augmented as the team works on additional features for C# 7.